### ISSUE TITLE
Plugins missing from system status report

### ISSUE THREAD

**Issue by WillBrubaker** [woocommerce/woocommerce#22056]
**Describe the bug**
Not all active plugins are listed under active plugins in the system status report

**To Reproduce**
Unfortunately, I am unable to identify what steps to take to reproduce this. This report is based on observations of system status reports provided by users when they request support. This seems to be fairly widespread.

As an example, see ticket: 1559643-zen - the system status report there does not list WooCommerce as an active plugin, but it obviously is active. It also lacks Stripe, but Stripe is also obviously active on the site in question. That system status report is the one I am posting here

**Expected behavior**
I would expect all my active plugins to be displayed.

**Isolating the problem (mark completed items with an [x]):**
- [ ] I have deactivated other plugins and confirmed this bug occurs when only WooCommerce plugin is active. 
- [x ] This bug happens with a default WordPress theme active, or [Storefront](https://woocommerce.com/storefront/).
- [ ] I can reproduce this bug consistently using the steps above.

**WordPress Environment**
<details>
```
### WordPress Environment ###

Home URL: https://joelysueburkhart.com 
Site URL: https://joelysueburkhart.com 
WC Version: 3.5.1 
Log Directory Writable: ✔ 
WP Version: 4.9.8 
WP Multisite: – 
WP Memory Limit: 256 MB 
WP Debug Mode: – 
WP Cron: ✔ 
Language: en_US 
External object cache: –

### Server Environment ###

Server Info: Apache/2.4.37 
PHP Version: 7.2.12 
PHP Post Max Size: 128 MB 
PHP Time Limit: 120 
PHP Max Input Vars: 2048 
cURL Version: 7.62.0 
OpenSSL/1.0.2p

SUHOSIN Installed: – 
MySQL Version: 5.5.5-10.2.19-MariaDB-log 
Max Upload Size: 100 MB 
Default Timezone is UTC: ✔ 
fsockopen/cURL: ✔ 
SoapClient: ✔ 
DOMDocument: ✔ 
GZip: ✔ 
Multibyte String: ✔ 
Remote Post: ✔ 
Remote Get: ✔

### Database ###

WC Database Version: 3.5.1 
WC Database Prefix: wp_ 
MaxMind GeoIP Database: ✔ 
Total Database Size: 68.75MB 
Database Data Size: 62.98MB 
Database Index Size: 5.77MB 
wp_woocommerce_sessions: Data: 0.19MB + Index: 0.02MB 
wp_woocommerce_api_keys: Data: 0.02MB + Index: 0.03MB 
wp_woocommerce_attribute_taxonomies: Data: 0.02MB + Index: 0.02MB 
wp_woocommerce_downloadable_product_permissions: Data: 0.02MB + Index: 0.05MB 
wp_woocommerce_order_items: Data: 0.02MB + Index: 0.02MB 
wp_woocommerce_order_itemmeta: Data: 0.02MB + Index: 0.03MB 
wp_woocommerce_tax_rates: Data: 0.02MB + Index: 0.06MB 
wp_woocommerce_tax_rate_locations: Data: 0.02MB + Index: 0.03MB 
wp_woocommerce_shipping_zones: Data: 0.02MB + Index: 0.00MB 
wp_woocommerce_shipping_zone_locations: Data: 0.02MB + Index: 0.03MB 
wp_woocommerce_shipping_zone_methods: Data: 0.02MB + Index: 0.00MB 
wp_woocommerce_payment_tokens: Data: 0.02MB + Index: 0.02MB 
wp_woocommerce_payment_tokenmeta: Data: 0.02MB + Index: 0.03MB 
wp_woocommerce_log: Data: 0.02MB + Index: 0.02MB 
wp_aryo_activity_log: Data: 0.02MB + Index: 0.00MB 
wp_commentmeta: Data: 2.52MB + Index: 0.52MB 
wp_comments: Data: 3.52MB + Index: 0.91MB 
wp_DLM_CATS: Data: 0.02MB + Index: 0.00MB 
wp_DLM_DOWNLOADS: Data: 0.02MB + Index: 0.00MB 
wp_DLM_FORMATS: Data: 0.02MB + Index: 0.00MB 
wp_DLM_LOG: Data: 0.02MB + Index: 0.00MB 
wp_DLM_META: Data: 0.02MB + Index: 0.00MB 
wp_DLM_STATS: Data: 0.02MB + Index: 0.00MB 
wp_failed_jobs: Data: 0.02MB + Index: 0.00MB 
wp_itsec_distributed_storage: Data: 2.02MB + Index: 0.02MB 
wp_itsec_fingerprints: Data: 0.02MB + Index: 0.03MB 
wp_itsec_geolocation_cache: Data: 0.02MB + Index: 0.03MB 
wp_itsec_lockouts: Data: 0.02MB + Index: 0.08MB 
wp_itsec_logs: Data: 8.02MB + Index: 0.39MB 
wp_itsec_temp: Data: 0.02MB + Index: 0.06MB 
wp_links: Data: 0.02MB + Index: 0.03MB 
wp_mailchimp_carts: Data: 0.02MB + Index: 0.00MB 
wp_options: Data: 1.31MB + Index: 0.06MB 
wp_postmeta: Data: 2.52MB + Index: 0.63MB 
wp_posts: Data: 34.55MB + Index: 1.05MB 
wp_queue: Data: 0.02MB + Index: 0.00MB 
wp_redirection_404: Data: 0.14MB + Index: 0.09MB 
wp_redirection_groups: Data: 0.02MB + Index: 0.03MB 
wp_redirection_items: Data: 0.02MB + Index: 0.08MB 
wp_redirection_logs: Data: 0.14MB + Index: 0.08MB 
wp_termmeta: Data: 0.02MB + Index: 0.03MB 
wp_terms: Data: 0.06MB + Index: 0.03MB 
wp_term_relationships: Data: 0.31MB + Index: 0.17MB 
wp_term_taxonomy: Data: 0.06MB + Index: 0.06MB 
wp_usermeta: Data: 0.05MB + Index: 0.03MB 
wp_users: Data: 0.02MB + Index: 0.05MB 
wp_wc_download_log: Data: 0.02MB + Index: 0.03MB 
wp_wc_webhooks: Data: 0.02MB + Index: 0.02MB 
wp_websitez_stats: Data: 0.09MB + Index: 0.02MB 
wp_wfBadLeechers: Data: 0.02MB + Index: 0.00MB 
wp_wfBlockedCommentLog: Data: 0.02MB + Index: 0.00MB 
wp_wfBlockedIPLog: Data: 0.06MB + Index: 0.00MB 
wp_wfBlocks: Data: 0.08MB + Index: 0.02MB 
wp_wfBlocks7: Data: 0.16MB + Index: 0.17MB 
wp_wfBlocksAdv: Data: 0.02MB + Index: 0.00MB 
wp_wfConfig: Data: 0.45MB + Index: 0.00MB 
wp_wfCrawlers: Data: 0.02MB + Index: 0.00MB 
wp_wfFileChanges: Data: 0.02MB + Index: 0.00MB 
wp_wfFileMods: Data: 2.52MB + Index: 0.00MB 
wp_wfHits: Data: 1.02MB + Index: 0.27MB 
wp_wfHoover: Data: 0.02MB + Index: 0.02MB 
wp_wfIssues: Data: 0.02MB + Index: 0.06MB 
wp_wfKnownFileList: Data: 1.52MB + Index: 0.00MB 
wp_wfLeechers: Data: 0.20MB + Index: 0.00MB 
wp_wfLiveTrafficHuman: Data: 0.02MB + Index: 0.02MB 
wp_wfLockedOut: Data: 0.02MB + Index: 0.00MB 
wp_wfLocs: Data: 0.02MB + Index: 0.00MB 
wp_wfLogins: Data: 0.09MB + Index: 0.03MB 
wp_wfNet404s: Data: 0.02MB + Index: 0.02MB 
wp_wfNotifications: Data: 0.05MB + Index: 0.00MB 
wp_wfPendingIssues: Data: 0.02MB + Index: 0.06MB 
wp_wfReverseCache: Data: 0.02MB + Index: 0.00MB 
wp_wfScanners: Data: 0.02MB + Index: 0.00MB 
wp_wfSNIPCache: Data: 0.02MB + Index: 0.05MB 
wp_wfStatus: Data: 0.14MB + Index: 0.09MB 
wp_wfThrottleLog: Data: 0.13MB + Index: 0.05MB 
wp_wfTrafficRates: Data: 0.02MB + Index: 0.00MB 
wp_wfVulnScanners: Data: 0.02MB + Index: 0.00MB 
wp_yoast_seo_links: Data: 0.02MB + Index: 0.02MB 
wp_yoast_seo_meta: Data: 0.02MB + Index: 0.00MB

### Post Type Counts ###

attachment: 533 
customize_changeset: 9 
feedback: 2 
jetpack_migration: 2 
jp_img_sitemap: 2 
jp_sitemap: 2 
jp_sitemap_master: 2 
nav_menu_item: 54 
oembed_cache: 1 
page: 90 
post: 1583 
product: 14 
revision: 3957 
shop_order: 1

### Security ###

Secure connection (HTTPS): ✔ 
Hide errors from visitors: ✔

### Active Plugins (12) ###

Query Monitor: by John Blackbourn & contributors – 3.1.1 
Akismet Anti-Spam: by Automattic – 4.1 
Activity Log: by Activity Log Team – 2.5.1 
iThemes Security: by iThemes – 7.2.0 
Disable Emojis (GDPR friendly): by Ryan Hellyer – 1.7.2 
Flexible Shipping: by WP Desk – 2.1.10 
Gutenberg: by Gutenberg Team – 4.5.1 
Jetpack by WordPress.com: by Automattic – 6.7 
Mailchimp for WooCommerce: by Mailchimp – 2.1.11 – Not tested with the active version of WooCommerce 
Really Simple SSL: by Rogier Lankhorst 
Mark Wolters – 3.1.2

Redirection: by John Godley – 3.6.3 
Storefront Blog Customiser: by WooCommerce – 1.2.5

### Settings ###

API Enabled: – 
Force SSL: – 
Currency: USD ($) 
Currency Position: left 
Thousand Separator: , 
Decimal Separator: . 
Number of Decimals: 2 
Taxonomies: Product Types: external (external) 
grouped (grouped) 
simple (simple) 
variable (variable)

Taxonomies: Product Visibility: exclude-from-catalog (exclude-from-catalog) 
exclude-from-search (exclude-from-search) 
featured (featured) 
outofstock (outofstock) 
rated-1 (rated-1) 
rated-2 (rated-2) 
rated-3 (rated-3) 
rated-4 (rated-4) 
rated-5 (rated-5)

### WC Pages ###

Shop base: #6769 - /shop/ 
Cart: #6770 - /cart/ 
Checkout: #6771 - /checkout/ 
My account: #6772 - /my-account/ 
Terms and conditions: ❌ Page not set

### Theme ###

Name: Storefront 
Version: 2.3.5 
Author URL: https://woocommerce.com/ 
Child Theme: ❌ – If you are modifying WooCommerce on a parent theme that you did not build personally we recommend using a child theme. See: How to create a child theme 
WooCommerce Support: ✔

### Templates ###

Overrides: – 
```
</details>

**Comment by mikkamp** [comment]
I don't have any steps to replicate this either. But I did see it happen on a site recently. After visiting the Plugins page and then going back to the status report it did show the missing plugins in the status report. So I wonder if it has anything to do with reading from a cached list of active plugins.

**Comment by harishankerr** [comment]
I don't have steps to replicate it myself, but I found this on a site, recently. The site clearly had Product Add-ons installed, but it was missing from the System Status report.

**Comment by mikkamp** [comment]
Since it seems to have been showing up after 3.5 could it be anything related to this PR: https://github.com/woocommerce/woocommerce/pull/21073

**Comment by dougaitken** [comment]
I can't replicate this but I often find even on my own site it take a hard refresh or two to get the full report. 

I just loaded my status page and it shows this:

```
### Active Plugins (4) ###

Akismet Anti-Spam: by Automattic – 4.1
Bookings Helper: by WooCommerce – 1.0.2 – Not tested with the active version of WooCommerce
Email Log: by Sudar – 2.2.5
Health Check & Troubleshooting: by The WordPress.org community – 1.2.1
```

Hard refresh gives this:

```
### Active Plugins (6) ###

Akismet Anti-Spam: by Automattic – 4.1
Bookings Helper: by WooCommerce – 1.0.2 – Not tested with the active version of WooCommerce
Email Log: by Sudar – 2.2.5
Health Check & Troubleshooting: by The WordPress.org community – 1.2.1
Homepage Control: by WooThemes – 2.0.3
Jetpack by WordPress.com: by Automattic – 6.7
```

FWIW, the active plugins changes each time I've come across this.

I have 53 active plugins on this site

It took refreshed to get the correct SSR (4, 6, 10, 41, then 53 active plugins)
<details>
`
### WordPress Environment ###

Home URL: https://doug.press
Site URL: https://doug.press
WC Version: 3.5.1
Log Directory Writable: ✔
WP Version: 4.9.8
WP Multisite: –
WP Memory Limit: 256 MB
WP Debug Mode: ✔
WP Cron: ✔
Language: en_GB
External object cache: ✔

### Server Environment ###

Server Info: nginx
PHP Version: 7.0.32 - We recommend using PHP version 7.2 or above for greater performance and security. How to update your PHP version
PHP Post Max Size: 100 MB
PHP Time Limit: 300
PHP Max Input Vars: 6144
cURL Version: 7.61.1
OpenSSL/1.0.1t

SUHOSIN Installed: –
MySQL Version: 5.5.5-10.1.17-MariaDB
Max Upload Size: 100 MB
Default Timezone is UTC: ✔
fsockopen/cURL: ✔
SoapClient: ✔
DOMDocument: ✔
GZip: ✔
Multibyte String: ✔
Remote Post: ✔
Remote Get: ✔

### Database ###

WC Database Version: 3.5.1
WC Database Prefix: wp_
MaxMind GeoIP Database: ✔
Total Database Size: 77.94MB
Database Data Size: 70.43MB
Database Index Size: 7.51MB
wp_woocommerce_sessions: Data: 14.02MB + Index: 0.27MB
wp_woocommerce_api_keys: Data: 0.02MB + Index: 0.03MB
wp_woocommerce_attribute_taxonomies: Data: 0.02MB + Index: 0.02MB
wp_woocommerce_downloadable_product_permissions: Data: 0.02MB + Index: 0.05MB
wp_woocommerce_order_items: Data: 0.06MB + Index: 0.02MB
wp_woocommerce_order_itemmeta: Data: 0.23MB + Index: 0.22MB
wp_woocommerce_tax_rates: Data: 0.02MB + Index: 0.06MB
wp_woocommerce_tax_rate_locations: Data: 0.02MB + Index: 0.05MB
wp_woocommerce_shipping_zones: Data: 0.02MB + Index: 0.00MB
wp_woocommerce_shipping_zone_locations: Data: 0.02MB + Index: 0.05MB
wp_woocommerce_shipping_zone_methods: Data: 0.02MB + Index: 0.00MB
wp_woocommerce_payment_tokens: Data: 0.02MB + Index: 0.02MB
wp_woocommerce_payment_tokenmeta: Data: 0.02MB + Index: 0.03MB
wp_woocommerce_log: Data: 0.02MB + Index: 0.02MB
wp_advanced_notifications: Data: 0.02MB + Index: 0.00MB
wp_advanced_notification_triggers: Data: 0.02MB + Index: 0.00MB
wp_commentmeta: Data: 0.02MB + Index: 0.03MB
wp_comments: Data: 0.33MB + Index: 0.39MB
wp_email_log: Data: 4.52MB + Index: 0.00MB
wp_followup_coupons: Data: 0.02MB + Index: 0.03MB
wp_followup_coupon_logs: Data: 0.02MB + Index: 0.05MB
wp_followup_customers: Data: 0.02MB + Index: 0.06MB
wp_followup_customer_carts: Data: 0.02MB + Index: 0.03MB
wp_followup_customer_notes: Data: 0.02MB + Index: 0.03MB
wp_followup_customer_orders: Data: 0.02MB + Index: 0.02MB
wp_followup_email_coupons: Data: 0.02MB + Index: 0.02MB
wp_followup_email_excludes: Data: 0.02MB + Index: 0.05MB
wp_followup_email_logs: Data: 0.02MB + Index: 0.05MB
wp_followup_email_orders: Data: 0.09MB + Index: 0.09MB
wp_followup_email_order_coupons: Data: 0.02MB + Index: 0.02MB
wp_followup_email_tracking: Data: 0.02MB + Index: 0.08MB
wp_followup_followup_history: Data: 0.08MB + Index: 0.05MB
wp_followup_order_categories: Data: 0.02MB + Index: 0.03MB
wp_followup_order_items: Data: 0.02MB + Index: 0.05MB
wp_followup_subscribers: Data: 0.02MB + Index: 0.03MB
wp_followup_subscribers_to_lists: Data: 0.02MB + Index: 0.03MB
wp_followup_subscriber_lists: Data: 0.02MB + Index: 0.03MB
wp_links: Data: 0.02MB + Index: 0.02MB
wp_options: Data: 28.34MB + Index: 0.28MB
wp_postmeta: Data: 10.25MB + Index: 3.41MB
wp_posts: Data: 2.23MB + Index: 0.31MB
wp_rg_form: Data: 0.02MB + Index: 0.00MB
wp_rg_form_meta: Data: 0.02MB + Index: 0.00MB
wp_rg_form_view: Data: 0.02MB + Index: 0.03MB
wp_rg_incomplete_submissions: Data: 0.02MB + Index: 0.02MB
wp_rg_lead: Data: 0.02MB + Index: 0.03MB
wp_rg_lead_detail: Data: 0.02MB + Index: 0.06MB
wp_rg_lead_detail_long: Data: 0.02MB + Index: 0.00MB
wp_rg_lead_meta: Data: 0.02MB + Index: 0.05MB
wp_rg_lead_notes: Data: 0.02MB + Index: 0.03MB
wp_termmeta: Data: 0.02MB + Index: 0.03MB
wp_terms: Data: 0.02MB + Index: 0.03MB
wp_term_relationships: Data: 0.06MB + Index: 0.02MB
wp_term_taxonomy: Data: 0.02MB + Index: 0.03MB
wp_usermeta: Data: 0.09MB + Index: 0.06MB
wp_users: Data: 0.02MB + Index: 0.05MB
wp_wc_booking_relationships: Data: 0.02MB + Index: 0.03MB
wp_wc_deposits_payment_plans: Data: 0.02MB + Index: 0.00MB
wp_wc_deposits_payment_plans_schedule: Data: 0.02MB + Index: 0.02MB
wp_wc_download_log: Data: 0.02MB + Index: 0.03MB
wp_wc_points_rewards_user_points: Data: 0.02MB + Index: 0.03MB
wp_wc_points_rewards_user_points_log: Data: 0.02MB + Index: 0.05MB
wp_wc_warranty_products: Data: 0.02MB + Index: 0.05MB
wp_wc_webhooks: Data: 0.02MB + Index: 0.02MB
wp_woocommerce_bundled_itemmeta: Data: 0.02MB + Index: 0.03MB
wp_woocommerce_bundled_items: Data: 0.02MB + Index: 0.03MB
wp_woocommerce_exported_csv_items: Data: 8.52MB + Index: 0.02MB
wp_woocommerce_per_product_shipping_rules: Data: 0.02MB + Index: 0.00MB
wp_woocommerce_pickup_locations_geodata: Data: 0.02MB + Index: 0.05MB
wp_woocommerce_shipping_table_rates: Data: 0.02MB + Index: 0.00MB
wp_woocommerce_software_activations: Data: 0.02MB + Index: 0.00MB
wp_woocommerce_software_licenses: Data: 0.02MB + Index: 0.00MB
wp_woocommerce_subscription_downloads: Data: 0.02MB + Index: 0.00MB
wp_wps_hit: Data: 0.02MB + Index: 0.14MB
wp_wps_index: Data: 0.25MB + Index: 0.38MB
wp_wps_key: Data: 0.06MB + Index: 0.02MB
wp_wps_object_type: Data: 0.02MB + Index: 0.08MB
wp_wps_query: Data: 0.02MB + Index: 0.02MB
wp_wps_uri: Data: 0.02MB + Index: 0.02MB
wp_wps_user_agent: Data: 0.02MB + Index: 0.02MB

### Post Type Counts ###

attachment: 198
bookable_person: 5
bookable_resource: 5
custom_css: 1
customize_changeset: 8
event_ticket: 6
feedback: 2
follow_up_email: 17
global_product_addon: 1
jp_img_sitemap: 2
jp_sitemap: 2
jp_sitemap_master: 2
jp_vid_sitemap: 2
nav_menu_item: 11
oembed_cache: 1
page: 27
post: 9
product: 116
product_variation: 235
revision: 234
scheduled-action: 153
shop_coupon: 30
shop_order: 170
shop_order_refund: 15
shop_subscription: 14
warranty_request: 1
wc_booking: 49
wc_membership_plan: 4
wc_pickup_location: 1
wc_user_membership: 2
wishlist: 1

### Security ###

Secure connection (HTTPS): ✔
Hide errors from visitors: ✔

### Active Plugins (53) ###

Akismet Anti-Spam: by Automattic – 4.1
Bookings Helper: by WooCommerce – 1.0.2 – Not tested with the active version of WooCommerce
Email Log: by Sudar – 2.2.5
Health Check & Troubleshooting: by The WordPress.org community – 1.2.1
Homepage Control: by WooThemes – 2.0.3
Jetpack by WordPress.com: by Automattic – 6.7
Storefront Footer Bar: by WooThemes – 1.0.4
Storefront Parallax Hero: by WooCommerce – 1.5.7
Storefront Powerpack: by WooCommerce – 1.4.12
Storefront Product Hero: by WooThemes – 1.2.13
WooCommerce Accommodation Bookings: by WooCommerce – 1.1.5
WooCommerce Payment Gateway Based Fees: by InoPlugs – 3.1.5 – Not tested with the active version of WooCommerce
WooCommerce Additional Variation Images: by WooCommerce – 1.7.14
WooCommerce Bookings: by Automattic – 1.12.2
WooCommerce Box Office: by WooCommerce – 1.1.11
WooCommerce Brands: by WooCommerce – 1.6.4
WooCommerce Chained Products: by WooCommerce – 2.8.3
WooCommerce Checkout Field Editor: by WooCommerce – 1.5.15
WooCommerce Composite Products: by SomewhereWarm – 3.14.5 – 3.14.6 is available
WooCommerce Conditional Shipping and Payments: by SomewhereWarm – 1.4.2
WooCommerce Customer/Order CSV Export: by SkyVerge – 4.6.0
WooCommerce Deposits: by Automattic – 1.4.7
WooCommerce Dynamic Pricing: by Lucas Stark – 3.1.10 – Not tested with the active version of WooCommerce
WooCommerce EU VAT Number: by WooCommerce – 2.3.11
Follow-Up Emails: by WooCommerce – 4.8.4
WooCommerce Force Sells: by WooCommerce – 1.1.17
WooCommerce PayPal Pro Hosted: by WooCommerce – 1.0.18
WooCommerce Stripe Gateway: by WooCommerce – 4.1.13
WooCommerce Memberships: by SkyVerge – 1.12.2
WooCommerce Min/Max Quantities: by WooCommerce – 2.4.5
WooCommerce Mix and Match: by Kathy Darling – 1.3.4
WooCommerce Name Your Price: by Kathy Darling – 2.9.3
WooCommerce One Page Checkout: by Prospress Inc. – 1.6.0 – Not tested with the active version of WooCommerce
WooCommerce Order Barcodes: by Automattic – 1.3.4
WooCommerce Photography: by WooCommerce – 1.0.15
WooCommerce Pre-Orders: by WooCommerce – 1.5.10
WooCommerce Product Add-ons: by WooCommerce – 3.0.4
WooCommerce Product Bundles - Min/Max Items: by SomewhereWarm – 1.3.4
WooCommerce Product Bundles: by SomewhereWarm – 5.8.0 – 5.8.1 is available
WooCommerce Product CSV Import Suite: by WooCommerce – 1.10.20
WooCommerce Product Enquiry Form: by WooCommerce – 1.2.7
WooCommerce Services: by Automattic – 1.17.1
WooCommerce Royal Mail: by WooCommerce – 2.5.11
WooCommerce Store Credit: by WooCommerce – 2.2.0
WooCommerce Subscription Downloads: by WooCommerce – 1.1.17
WooCommerce Subscriptions: by Prospress Inc. – 2.4.4
WooCommerce Table Rate Shipping: by WooCommerce – 3.0.12
WooCommerce Variation Swatches and Photos: by Lucas Stark – 3.0.10
WooCommerce Warranty Requests: by WooCommerce – 1.8.13
WooCommerce: by Automattic – 3.5.1
WordPress Beta Tester: by Peter Westwood – 1.2.3
WordPress Importer: by wordpressdotorg – 0.6.4
WP Crontrol: by John Blackbourn & contributors – 1.6.2

### Settings ###

API Enabled: ✔
Force SSL: ✔
Currency: GBP (£)
Currency Position: left
Thousand Separator: ,
Decimal Separator: .
Number of Decimals: 2
Taxonomies: Product Types: accommodation-booking (accommodation-booking)
booking (booking)
bundle (bundle)
composite (composite)
external (external)
grouped (grouped)
mix and match (mix-and-match)
photography (photography)
simple (simple)
subscription (subscription)
variable (variable)
variable subscription (variable-subscription)

Taxonomies: Product Visibility: exclude-from-catalog (exclude-from-catalog)
exclude-from-search (exclude-from-search)
featured (featured)
outofstock (outofstock)
rated-1 (rated-1)
rated-2 (rated-2)
rated-3 (rated-3)
rated-4 (rated-4)
rated-5 (rated-5)


### WC Pages ###

Shop base: #3 - /shop/
Basket: #4 - /cart/
Checkout: #5 - /checkout/
My account: #6 - /my-account/
Terms and conditions: #2385 - /privacy-policy/

### Theme ###

Name: Storefront
Version: 2.3.5
Author URL: https://woocommerce.com/
Child Theme: ❌ – If you are modifying WooCommerce on a parent theme that you did not build
personally we recommend using a child theme. See: How to create a child theme

WooCommerce Support: ✔

### Templates ###

Overrides: –

### Memberships ###

Restriction mode:: Redirect to page
Public content:: Products: 1 item
Excluded from discounts:: –
Membership plans:: 4 (2 published)
Everything (draft):: Members: 0Access method: Manual assignment onlyAccess length: Unlimited Subscription: No
Certification Class (draft):: Members: 0Access method: Manual assignment onlyAccess length: Unlimited Subscription: No
submem:: Members: 1Access method: Product(s) purchaseAccess length: Unlimited Subscription: No No access-granting products are purchasable
Test Memberships:: Members: 1Access method: Manual assignment onlyAccess length: Unlimited Subscription: No

### Subscriptions ###

WCS_DEBUG: ✔ No
Subscriptions Mode: ❌ Staging
Subscriptions Live URL: https://dougpress.mystagingwebsite.com
Subscription Statuses: wc-expired: 2
wc-on-hold: 3
wc-cancelled: 9

WooCommerce Account Connected: ✔ Yes
Active Product Key: ❌ No
Report Cache Enabled: ✔ Yes
Cache Update Failures: ✔ 0 failure

### Store Setup ###

Country / State: United Kingdom (UK)

### Subscriptions by Payment Gateway ###

other: wc-cancelled: 3
Stripe: wc-cancelled: 6
wc-expired: 2
wc-on-hold: 3


### Payment Gateway Support ###

Stripe: products
refunds
tokenization
add_payment_method
subscriptions
subscription_cancellation
subscription_suspension
subscription_reactivation
subscription_amount_changes
subscription_date_changes
subscription_payment_method_change
subscription_payment_method_change_customer
subscription_payment_method_change_admin
multiple_subscriptions
pre-orders

Cash on delivery: products

### Mix and Match Products ###

Template Overrides: –

### Composite Products ###

Database Version: 3.14.5
Template Overrides: –

### Product Bundles ###

Database Version: 5.8.0
Template Overrides: –

### Customer/Order CSV Export ###

Order Export Format: custom
Customer Export Format: default
Automatically Export Orders: disabled
Automatically Export Customers: disabled
`
</details>

**Comment by claudiulodro** [comment]
Should be fixed in #22057
